docs/setup/install-new.md

```diff
@@ -47,23 +49,60 @@ Click on the **Deploy to Azure** icon below to begin the infrastructure deployme
 - [Public IP Address](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-public-ip-address)
 - [Managed Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview), which will be used by [AAD Pod Identity](https://github.com/Azure/aad-pod-identity/blob/master/README.md)
 
-#### Important
-Please use the `appId`, `objectId`, and `password` values from the `az` commands above and
-paste them in the corresponding ARM template fields:
-  - paste the `appId` vaule in the `Aks Service Principal App Id` template field
-  - paste the `password` value in the `Aks Service Principal Client Secret` field
-  - paste the `objectId` value in the `Aks Service Principal Object Id` field
+1. Download the ARM template and modify the template as needed.
+    ```bash
+    wget https://raw.githubusercontent.com/Azure/application-gateway-kubernetes-ingress/master/deploy/azuredeploy.json -O template.json
+    ```
 
-Note: To deploy an **RBAC** enabled cluster, set the `aksEnabledRBAC` field to `true`
+1. Create the parameter file.
+    ```bash
+    cat <<EOF > parameters.json
+    {
+      "aksServicePrincipalAppId": { "value": "$appId" },
+      "aksServicePrincipalClientSecret": { "value": "$password" },
+      "aksServicePrincipalObjectId": { "value": "$objectId" },
+      "aksEnableRBAC": { "value": false }
+    }
+    EOF
+    ```
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
-<img src="http://azuredeploy.net/deploybutton.png"/>
-</a>
+    #### Important
+    Please use the `appId`, `objectId`, and `password` values from the `az` commands above and
+    paste them in the corresponding ARM template fields:
+      - paste the `appId` vaule in the `Aks Service Principal App Id` template field
+      - paste the `password` value in the `Aks Service Principal Client Secret` field
+      - paste the `objectId` value in the `Aks Service Principal Object Id` field
 
-Navigate to the deployment output and record the parameters:
-[Azure portal](https://portal.azure.com/): `Home -> *resource group* -> Deployments -> *new deployment* -> Outputs`)
+    Note: To deploy an **RBAC** enabled cluster, set the `aksEnabledRBAC` field to `true`
+
+1. Deploy the ARM template.
+    ```bash
+    resourceGroupName="MyResourceGroup"
+    location="westus2"
+    deploymentName="ingress-appgw"
+
+    # create a resource group
+    az group create -n $resourceGroupName -l $location
+
+    # modify the template as needed
+    az group deployment create \
+            -g $resourceGroupName \
+            -n $deploymentName \
+            --template-file template.json \
+            --parameters ./parameters.json
+    ```
+
+1. Save the output from the deployment
```

Comment:
We might want to say something like - download the output of the deployment process into a file named xxx

Do we have to wait a certain time for the deployment to finish? Should we mention that the deployment itself is a blocking operation for the shell command (is it)?
Should we provide a way for the customer to use CLI command to check the status of the deployment?